frontend/src/metabase/lib/formatting/date.tsx

```diff
@@ -690,16 +690,18 @@ function replaceDateFormatNames(format: string, options: OptionsType) {
 }
 
 function formatDateTimeWithFormats(
-  value: number,
+  value: number | Moment,
```

Comment:
This is not needed for the fix, but I wanted to update the type to reflect the fact that this function is called with `value` as a `Moment` object.